panopticum/models.py

```diff
@@ -276,9 +276,9 @@ class Requirement(models.Model):
     title = models.CharField(max_length=30, unique=True)  # backup, logging storage
     description = models.TextField(max_length=1024)  # that requirements about ...
 
-
     def __unicode__(self):
-        return f"{self.__class__.__name__}: {self.title}"
+        sets = self.sets.all()
```

Comment:
```self.sets.all().first()``` will fetch one entry instead all.
```self.sets.exists()``` will optimally check that query set not empty. https://docs.djangoproject.com/en/dev/ref/models/querysets/#exists

Reply:
ok